.circleci/config.yml

```diff
@@ -230,8 +230,8 @@ jobs:
           command: ./gradlew assembleSelfSignedRelease
 
       - run:
-          name: Check APK size hasn't increased by more than 100kb
-          command: if [ $(ls -l collect_app/build/outputs/apk/selfSignedRelease/*.apk | awk '{print $5}') -gt 10734563 ]; then exit 1; fi
+          name: Check APK size hasn't increased too much
+          command: if [ $(ls -l collect_app/build/outputs/apk/selfSignedRelease/*.apk | awk '{print $5}') -gt 10000000 ]; then exit 1; fi
```

Comment:
Did you experiment with going any lower here? I think I'm more focused on this warning us about any increase (so we can consider it) as opposed to just going over a certain size.

Reply:
Unfortunately, I had to increase it after rebasing onto master because of https://github.com/getodk/collect/pull/5802#discussion_r1386595876 so now it's 10.2mb still less than in v2023.3.0